config/locales/en.yml

```diff
@@ -25,6 +25,9 @@ en:
   components:
     attachment:
       opendocument_html: "This file is in an <a href='https://www.gov.uk/guidance/open-document-format-odf-guidance-for-uk-government/overview-of-productivity-software' target=%{target} class='govuk-link'>OpenDocument</a> format"
+      request_format_text: "This file may not be suitable for users of assistive technology."
+      request_format_cta: "Request a different format"
+      request_format_details_html: "If you use assistive technology and need a version of this document in a more accessible format, please email <a href='mailto:%{alternative_format_contact_email}' target='_blank' class='govuk-link'>%{alternative_format_contact_email}</a>. Please tell us what format you need. It will help us if you say what assistive technology you use."
```

Comment:
Should we make the target configurable, like we do for the opendocument guidance?

Reply:
As discussed, I think we'd want to enforce this to be blank since to make sure users are not loosing the context (the page they're on at the point they request another format).